atlasdb-client/src/integrationInput/java/com/palantir/atlasdb/table/description/generated/SchemaApiTestTable.java

```diff
@@ -533,25 +533,25 @@ public void putColumn1UnlessExists(Map<SchemaApiTestRow, Long> map) {
         putUnlessExists(Multimaps.forMap(toPut));
     }
 
-    public void putColumn2(SchemaApiTestRow row, String value) {
+    public void putColumn2(SchemaApiTestRow row, com.palantir.atlasdb.table.description.StringValue value) {
         put(ImmutableMultimap.of(row, Column2.of(value)));
     }
 
-    public void putColumn2(Map<SchemaApiTestRow, String> map) {
+    public void putColumn2(Map<SchemaApiTestRow, com.palantir.atlasdb.table.description.StringValue> map) {
```

Comment:
hmm, need to get rid of these package qualifiers

Reply:
actually i don't think this behavior changed for the v1 generated code, so leaving as is